src/Microsoft.Graph.Auth/Microsoft.Graph.Auth.csproj

```diff
@@ -30,7 +30,7 @@ Release Summary
     <AssemblyOriginatorKeyFile>35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Graph.Core" Version="1.15.0" />
+    <PackageReference Include="Microsoft.Graph.Core" Version="1.*" />
```

Comment:
Can you bump the VersionPrefix to 0.2.0? We released a 0.1.0-preview (takes precedence), and then 0.1.0-preview.x which didn't follow semver 2.0.0, so if we bump the VersionPrefix 0.2.0, we can get back on the semver 2.0.0 path.